spec/live/ticket_spec.rb

```diff
@@ -27,6 +27,39 @@ def valid_attributes
   it_should_be_readable agent, :ccd_tickets, create: true
   it_should_be_readable organization, :tickets
 
+  describe "#create" do
+    context "when passing large objects as parameters" do
+      let(:requester) { client.users.search(query: 'role:end-user').reject { |user| user.photo.nil? }.first }
+      let(:organization) { client.organizations.sample }
+      let(:ticket_parameters) do
+        {
+          subject: 'live spec subject',
+          description: 'live spec description',
+          requester: requester,
+          organization: organization
+        }
+      end
+
+      before do
+        VCR.use_cassette("ticket_create_with_objects") do
+          @ticket = ZendeskAPI::Ticket.create(client, ticket_parameters)
+        end
+      end
+
+      it 'should be creatable' do
+        expect(requester).to_not be_nil
+        expect(@ticket).to_not be_nil
+        expect(@ticket.send(:id)).to_not be_nil
+      end
```

Comment:
Using "should" in descriptions is not good practise, see https://www.betterspecs.org/#should 

Why are we using `send` here? That's not ideal for two reasons:

* `send` is for calling private methods
* even using `public_send` would be kinda confusing where `@ticket.id` would suffice

Is there any reason to use an instance var instead of a let here? Maybe something to do with `before and after`? See https://www.betterspecs.org/#let

Testing `not_nil` is a very weak test, can we add an expectation on the parameters being there? Also, when possible, it would be clearer to avoid negative statements, ref: https://www.obsidiansecurity.com/blog/why-to-stop-writing-negative-code/


```suggestion
      it 'creates the ticket' do
        expect(requester).to_not be_nil
        
        expect(ticket.id).to be_present
        expect(ticket.requested.reload!.photo.size).to be 123
      end
```

Reply:
thanks @ecoologic, I have pushed some changes. 
I couldn't use `be_present` in this case. it's added by Rails, hence the `not_nil`.
and yes the ticket needs to be an instance var because it gets created in the before block within that VCR code.